.cirrus.yml

```diff
@@ -708,6 +703,30 @@ podman_machine_task:
     always: *int_logs_artifacts
 
 
+podman_machine_aarch64_task:
+    name: *std_name_fmt
+    alias: podman_machine_aarch64
+    only_if: *not_tag_branch_build_docs
+    depends_on:
+        - build
```

Comment:
I'm really sorry, I know this has been pending so long, but DAE think that this should be `build_aarch64` instead of `build`, and should also include `validate_aarch64` since nothing else depends on it?

Reply:
Good catch, ya having that wrong has bitten us before (in different contexts).  I'm okay if that happens in a followup PR.

Reply:
Followup is OK, I guess. There are a LOT of unnecessary and confusing and wrong dependencies in the yml, now that I look at it. For instance, `podman_machine_task` should depend on `validate`, not on `build` (that's the whole source of this copy-paste error).

Reply:
i guess i messed  up recently in copy-pasta.

Reply:
fixing this in the next push.

Reply:
Nah, it's a perfectly understandable oversight. This cirrus-yml stuff is too complicated. And, I see no reason to push until the actual test failures (ssh? VM?) are fixed.